src/testing/README.md

```diff
@@ -230,3 +232,46 @@ const render = h.getRender();
 // Returns the result of the render for the index provided
 h.getRender(1);
 ```
+
+#### `harness.select`
+
+`harness.select` returns a list of DNodes, using the latest render, that match the provided selector. This can be useful if you need access to a widget's properties.
+
+```typescript
+select(selector: string);
+```
+
+-   `selector`: The selector to use while determining matching DNodes
```

Comment:
mention it returns an array? and in the API above?

Reply:
Updated the select api to be `select(selector: string): DNode[]`